uni/lib/model/providers/session_provider.dart

```diff
@@ -54,9 +55,18 @@ class SessionProvider extends StateProviderNotifier {
 
         updateStatus(RequestStatus.successful);
       } else {
+        //Check if password expired
```

Comment:
Avoid comments if the code is self-explanatory (like here IMO)

Reply:
Right. I will keep the "// No internet connection or server down" because that case isn't clear